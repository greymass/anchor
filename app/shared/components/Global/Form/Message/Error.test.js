import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import GlobalFormMessageError from './Error';

const mockProps = {
  chainSymbol: 'EOS',
  error: 'there is an error!',
  onClose: () => null,
  style: { marginTop: 10 },
  t: () => null,
};

describe('GlobalFormMessageError', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<GlobalFormMessageError {...mockProps} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
