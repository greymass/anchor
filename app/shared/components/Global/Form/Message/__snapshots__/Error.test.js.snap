// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalFormMessageError renders correctly 1`] = `
<I18n
  bindI18n="languageChanged loaded"
  bindStore="added removed"
  chainSymbol="EOS"
  error="there is an error!"
  ns={
    Array [
      "global",
    ]
  }
  nsMode="default"
  omitBoundRerender={true}
  onClose={[Function]}
  style={
    Object {
      "marginTop": 10,
    }
  }
  t={[Function]}
  translateFuncName="t"
  usePureComponent={false}
  wait={false}
  withRef={false}
>
  <Component />
</I18n>
`;
